metricflow/dataflow/dataflow_plan.py

```diff
@@ -1248,6 +1253,36 @@ def with_new_parents(self, new_parent_nodes: Sequence[BaseOutput]) -> ConstrainT
         )
 
 
+class MinMaxNode(BaseOutput):
+    """Calculate the min and max of a single instance data set."""
+
+    def __init__(self, parent_node: BaseOutput) -> None:  # noqa: D
+        self._parent_node = parent_node
+        super().__init__(node_id=self.create_unique_id(), parent_nodes=[parent_node])
+
+    @classmethod
+    def id_prefix(cls) -> str:  # noqa: D
+        return DATAFLOW_NODE_MIN_MAX_ID_PREFIX
```

Comment:
@plypaul is this still ok given your other changes to an enumerated ID prefix type?